.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release kak-lsp
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Oh well, that's a security issue, one could read the API key from the environment.
I only used this for debugging, will remove.

Reply:
Nvm, of course it's needed for creating the release draft, and uploading the binaries - I'll just avoid exposing it to the test script.